src/utils/paOnAuthorizationSuccess.js

```diff
@@ -0,0 +1,61 @@
+import paOnAuthorizationSuccessValidator from '../validators/paOnAuthorizationSuccess';
+
+const credentials = 'include';
+
+const getParameters = (response) => {
+  const result = [];
+  Object.keys(response).forEach(key => result.push(key + '=' + response[key]))
+  return result.join('&');
+};
+
+const postHeaders = () => {
+  const headers = new Headers();
+
+  headers.append("Content-Type", "application/x-www-form-urlencoded");
+  return headers;
+};
+
+const postOidcAuthnResponse = (oidcAuthnResponseEndpoint, callback) => (response) => {
+  const headers = postHeaders();
+  const body = getParameters(response);
+
+  const options = {
+    headers,
+    body,
+    credentials,
+    method: 'POST',
+  };
+
+  fetch(oidcAuthnResponseEndpoint, options)
+    .then(resp => callback(null, resp))
+    .catch(err => callback(err));
+};
+
+const getOptions = {
+  credentials,
+  method: 'GET'
+};
+
+const getOidcAuthnResponse = (oidcAuthnResponseEndpoint, callback) => (response) => {
+  const parameters = getParameters(response);
+
+  fetch(`${oidcAuthnResponseEndpoint}?${parameters}`, getOptions)
```

Comment:
Do we need to URL encode parameter values for the GET request?

Reply:
I'm not sure. I don't see anywhere in the widget that they are being encoded. The response argument would be the `authorizeResponse` field of the API response. I believe it is possible for that field to contain values that need to be encoded. Since `encodeURI()` does not encode "&" or "=", would `fetch(`${oidcAuthnResponseEndpoint}?${encodeURI(parameters)}`, getOptions)` work?

Reply:
Seems good to me. @khosravian agreed?

Reply:
+1

Reply:
Okay, I'll get another commit in with that change

Reply:
Thanks!